examples/sklearn_logistic_mod/train.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
```

Comment:
Any new examples need to be registered to execute as part of the Examples CI here: https://github.com/mlflow/mlflow/blob/101ad6e8eb383c769178df0df83d1d2a1cea6b4a/tests/examples/test_examples.py#L92-L94
. Do we need this additional test or can a signature example be added to examples/sklearn_logistic_regression/train.py? 
It might also be worthwhile to add a load from the logged (saved) location to run the score inference again to demonstrate the loading of the pyfunc flavor with the type-validated signature for a .csv file.